crates/cli/tests/reference/enums.js

```diff
@@ -78,9 +79,16 @@ Yellow:1,"1":"Yellow",
  * Red as a rose.
  */
 Red:2,"2":"Red", });
-
+/**
+ * @enum {0 | 1 | 42 | 43}
+ */
 export const ImplicitDiscriminant = Object.freeze({ A:0,"0":"A",B:1,"1":"B",C:42,"42":"C",D:43,"43":"D", });
 
+/**
+ * The name of a color.
+ * @typedef {"green" | "yellow" | "red"} ColorName
```

Comment:
Does this get registered even though the user only gets to use and see `get_name()`?